test/library/cluster_operator.go

```
@@ -14,6 +14,30 @@ import (
 	clusteroperatorhelpers "github.com/openshift/library-go/pkg/config/clusteroperator/v1helpers"
 )
 
+// WaitForClusterOperatorAvailableNotProgressingNotDegraded waits for a ClusterOperator to report
+// status as active, not progressing, and not failing.
+func WaitForClusterOperatorAvailableNotProgressingNotDegraded(t testing.TB, client configclient.ConfigV1Interface, name string) error {
+	err := wait.Poll(WaitPollInterval, WaitPollTimeout, func() (bool, error) {
```

golangci-lint (2.11.4)

[error] SA1019: wait.Poll is deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. 

(staticcheck)